models/enums.py

```diff
@@ -52,6 +52,7 @@ class ScenarioAttribute(Enum):
     YEAR_LEVEL = 6
     TIMESLOT_AVAILABILITY = 7
     PROJECT_PREFERENCES = 8
+    SOCIAL = 9
```

Comment:
I'm not which attribute this is supposed to denote.

In general, ScenaroAttribute is basically just a way for us to give names to attribute ids we have "reserved" for these meanings. Think of each one as being something "about" a student.

So students have genders, year levels, project preferences, etc. This list should only include scenario attributes we aim to run experiments on.

Note: If you meant this to mean the student's social preferences, we actually specify that in a different way (not an attribute, check models/student.py and note the relationships field). Also you would need to use the PreferenceGoal with the "social" setting instead